src/System.Net.HttpListener/tests/HttpRequestStreamTests.cs

```diff
@@ -313,7 +313,6 @@ public void Dispose()
         }
 
         [Fact]
-        [ActiveIssue(18128, platforms: TestPlatforms.AnyUnix)] // NotSupportedException thrown instead of InvalidOperationException
```

Comment:
I'm just confused because the only change to the src being made in this commit is the change to FlushAsync, which shouldn't have any affect on a NotSupportedException vs InvalidOperationException.  But maybe the comment was stale and that NSE vs IOE issue was previously fixed.  Regardless, if it's passing now, great.